Default username in github statistic commands

CaffeineDuck opened the pull request:
Fixed #15  

Now there is default username in .githubstats command!

Commit:
Fixed issue#15

Commit:
Full Fix for issue 315 with succesful test

Commit:
Fixed black formatting in github.py

FalseDev reviewed (changes requested):
A few changes to finish off:
 - Create a method called `get_gh_user` that takes the ctx to return the user (so its reusable)
 - Add this to `ghlangs` command as well
 - Might be simpler to use `username = username or self.get_gh_user(ctx)` as one line in both the commands

Commit:
Added a get_gh_user function;
Made defualt username avilable in ghlangs command;
Completely Fixed Issue #15 .

Samrid-Pandit commented:
> A few changes to finish off:
> 
> * Create a method called `get_gh_user` that takes the ctx to return the user (so its reusable)
> * Add this to `ghlangs` command as well
> * Might be simpler to use `username = username or self.get_gh_user(ctx)` as one line in both the commands

Changes are made!

FalseDev reviewed (approved):
This is perfect! Thank you for contributing!
